packages/synapse-react-client/src/lib/containers/DialogBase.tsx

```diff
@@ -39,8 +39,7 @@ export type DialogBaseProps = {
   className?: string
   onCancel: () => void
   hasCloseButton?: boolean
-  helpMarkdown?: string
-  helpUrl?: string
+  titleHelpPopoverProps?: HelpPopoverProps
```

Comment:
Allow DialogBase to handle all HelpPopoverProps so `placement` can be specified for the EntityFinderModal within DatasetItemsEditor